src/scenes/ModulesRouter/ModuleView.tsx

```diff
@@ -0,0 +1,169 @@
+          {label}
+        </Typography>
+        <Typography marginBottom={theme.spacing(1)}>{value}</Typography>
+        {onClickButton && <Button label='More Info' onClick={onClickButton} priority='secondary' />}
+      </Box>
+    </Grid>
+  );
+};
+
+const ModuleContentSection = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <Box
+      sx={{
+        flexDirection: 'column',
+        marginBottom: '16px',
+      }}
+    >
+      {children}
+    </Box>
+  );
+};
+
+const ModuleContentView = () => {
+  const dispatch = useAppDispatch();
+  const { activeLocale } = useLocalization();
+  const theme = useTheme();
+  const { goToModuleEvent } = useNavigateTo();
+  const pathParams = useParams<{ moduleId: string; projectId: string }>();
+  const moduleId = Number(pathParams.moduleId);
+  const projectId = Number(pathParams.projectId);
+  const module = useAppSelector(selectModule(moduleId));
+
+  const crumbs: Crumb[] = useMemo(
+    () => [
+      {
+        name: activeLocale ? strings.ALL_MODULES : '',
+        to: APP_PATHS.MODULES_FOR_PROJECT.replace(':projectId', `${projectId}`),
+      },
+    ],
+    [activeLocale, projectId]
+  );
+
+  useEffect(() => {
+    void dispatch(requestGetModule(moduleId));
+  }, [dispatch, moduleId]);
+
+  return (
+    <PageWithModuleTimeline crumbs={crumbs} hierarchicalCrumbs={false} title={<ModuleViewTitle module={module} />}>
+      <Card
+        sx={{
+          borderRadius: '24px',
+          boxShadow: 'none',
+          display: 'flex',
+          flexDirection: 'column',
+          flexGrow: 1,
+          marginBottom: theme.spacing(3),
+          padding: `${theme.spacing(2)} ${theme.spacing(1)}`,
+        }}
+      >
+        {module && (
+          <Grid container spacing={theme.spacing(1)}>
+            <Grid item xs style={{ flexGrow: 1, padding: `${theme.spacing(1)} ${theme.spacing(3)}` }}>
+              <ModuleContentSection>
+                <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={500}>
+                  {strings.formatString(strings.MODULE_NAME_OVERVIEW, module.name)}
+                </Typography>
+                <Typography fontSize={'24px'} lineHeight={'32px'} fontWeight={600}>
+                  {module.title}
+                </Typography>
+              </ModuleContentSection>
+
+              <ModuleContentSection>
+                <Box dangerouslySetInnerHTML={{ __html: module.description || '' }} />
+              </ModuleContentSection>
+
+              {module.contents.length && (
+                <>
+                  <ModuleContentSection>
+                    <Typography fontSize={'20px'} lineHeight={'28px'} fontWeight={600}>
+                      {strings.THIS_MODULE_CONTAINS}
+                    </Typography>
+                  </ModuleContentSection>
+
+                  {module.contents.map((content) => (
+                    <ModuleContentSection key={content.id}>
+                      <Link fontSize='16px' to='#'>
```

Comment:
Will set a value in the `to` prop for `Links` once I have a chance to create the route and placeholder page(s). What route(s) do we think would make sense these content items?

Reply:
We do need to link to deliverables or contents depending on the payload:

for deliverables: `/deliverables/:deliverableId/submissions/:projectId`

for contents, I left a comment on the routes above, hopefully it makes sense there.